VideoLocker/src/main/java/org/edx/mobile/http/ThrottlingInterceptor.java

```diff
@@ -38,5 +40,7 @@ public void intercept(RequestFacade request) {
         finally {
             requestLock.unlock();
         }
+
+        return chain.proceed(chain.request());
     }
 }
```

Comment:
I'm going to consider that out of scope here, but I'll see if there's any tool to automatically add newlines at the end of checked in source files, and open a dedicated pull request for that to get rid of this annoyance.